### PR TITLE
badusb: show script errors on screen

### PR DESCRIPTION
**Pull request by CromFr**
# What's new

- Display a precise error message on screen when there is a script error

# Verification 

You can try running the following scripts:

```
DELAY hellp
```
triggers an "Invalid number" message:
image

---
```
CTRL d

CTRL d
```
triggers a "Forbidden empty line" message:
image

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Comment by skotopes** [comment]
@CromFr we've freed-up enough space to accept your PR.
May I ask you to merge dev?

**Comment by CromFr** [comment]
@skotopes The PR has been rebased on dev.

The example with the forbidden empty line does not work anymore. I suppose somebody changed the script parser to allow empty lines, because empty lines are allowed on the dev branch

File: applications/main/bad_usb/bad_usb_script.c [hunk 1]
```diff
@@ -426,10 +454,20 @@ static int32_t ducky_script_execute_next(BadUsbScript* bad_usb, File* script_fil
...
-                delay_val = ducky_parse_line(bad_usb, bad_usb->line);
+                delay_val = ducky_parse_line(
+                    bad_usb, bad_usb->line, bad_usb->st.error, sizeof(bad_usb->st.error));
+
                 if(delay_val < 0) {
                     bad_usb->st.error_line = bad_usb->st.line_cur;
-                    FURI_LOG_E(WORKER_TAG, "Unknown command at line %u", bad_usb->st.line_cur);
+                    if(delay_val == SCRIPT_STATE_NEXT_LINE) {
+                        snprintf(
+                            bad_usb->st.error, sizeof(bad_usb->st.error), "Forbidden empty line");
```
Comment (nminaylov) [inline applications/main/bad_usb/bad_usb_script.c]: Empty lines are allowed, so no error is required here

Comment (CromFr) [reply applications/main/bad_usb/bad_usb_script.c]: :+1: fixed

**Review by skotopes — APPROVED** [review]

**Pull request merged**